orchard-ws/app/services/OrchardSystemService.scala

```diff
@@ -32,16 +32,18 @@ class OrchardSystemService @Inject() (
     j <- conf.getOptional[Double]("orchard.system.restart-jitter-probability")
   } yield (a, b, j)
 
+  private val checkProgressDelay = conf.get[Int]("com.salesforce.mce.orchard.activity.checkProgressDelayInMinutes") * 1.minutes
```

Comment:
@realstraw looks right to you?

Reply:
yes, except moving the parsing logic to `OrchardSettings` instead